high_card.rb

```diff
@@ -1,5 +1,31 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
 players = []
+deck = []
+hand = []
+
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank, suit])
+  end
+end
+
+shuffled = deck.shuffle
+
+puts players.length.to_s + " players so far. Enter a player name, or type 'play'"
+players << gets.chomp
+
+2.times do |cards|
+  hand.push(shuffled[cards])
+end
+
+puts "Ok, " + players.to_s + " gets " + hand[0].to_s + " and the computer gets " + hand[1].to_s
```

Comment:
couldn't you use a for each loop on players and print out the name and hand that way?